src/main/java/org/broadinstitute/consent/http/mail/freemarker/DatasetDeniedModel.java

```diff
@@ -5,11 +5,13 @@ public class DatasetDeniedModel {
   private String dataSubmitterName;
   private String datasetName;
   private String dacName;
+  private String dacEmail;
```

Comment:
It looks like these fields can be declared as `final`, since they're not modified once the object is constructed. If they were, then this could become a `record`:
```
public record DatasetDeniedModel(String dataSubmitterName, String datasetName, String dacName) {
}
```

Reply:
I like the idea of moving to record classes, this is a pretty old part of the code base.